service/raft/group0_state_machine.hh

```diff
@@ -8,54 +8,13 @@
 #pragma once
 
 #include "raft/raft.hh"
-#include "utils/UUID_gen.hh"
```

Comment:
Please submit this commit separately, it shouldn't wait for the entire series.

Reply:
Reducing the dependencies is an independent goal. I wish I could agree with you but the maintainers avoid cherry-picking today, if only to let CI pass before queuing. Consider this from the calendar and extra overhead point of view: a commit like this one could be queued within days, whereas it is going to take at least a week or two to get all the reviews on the entire series. I am all in favor of saving you and others from routine work, but here it seems not extracting this will increase routine for everyone.

Reply:
After some thought I don't think I will include this commit at all - if we aim to make the dependency graph acyclic, when group 0 client will depend on group 0 state machine, but not vice versa.